src/ucp/tag/eager_rcv.c

```diff
@@ -370,27 +335,23 @@ ucp_tag_offload_eager_first_handler(ucp_worker_h worker, void *data,
         return UCS_OK;
     }
 
+    /* Take data from rdesc, because it might be copied to new address */
+    priv                                 = (ucp_offload_first_desc_t*)(rdesc + 1);
```

Comment:
ucp_offload_first_desc_t - type variable should be called the same in all places.
currently it's either priv or first_hdr

Reply:
renamed